crates/nu-std/tests/run.nu

```diff
@@ -114,18 +117,32 @@ def main [
             log info $"Running tests in ($module.name)"
             $module.tests | each {|test|
                 log debug $"Running test ($test.name)"
-                nu -c $'
-                    use ($test.file) ($test.name)
+                let context_setup = if $test.setup {
+                    $"use `($test.file)` setup; let context = \(setup\)"
+                } else {
+                    "let context = {}"
+                }
+                let context_teardown = if $test.teardown {
+                    $"use `($test.file)` teardown; $context | teardown"
+                } else {
+                    ""
+                }
+                let nu_script = $'
+                    ($context_setup)
+                    use `($test.file)` ($test.name)
                     try {
-                        ($test.name)
+                        $context | ($test.name)
+                        ($context_teardown)
                     } catch { |err|
+                        ($context_teardown)
                         if $err.msg == "ASSERT:SKIP" {
                             exit 2
                         } else {
                             $err | get raw
                         }
                     }
                 '
+                nu -c $nu_script
```

Comment:
i'd just add some newlines betwee the `let`s, really minor thing

Reply:
Added.